packages/nodes-base/nodes/Google/Gmail/EmailDescriptions.ts

```diff
@@ -0,0 +1,251 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const emailOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'email',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Create Draft',
+				value: 'create',
+				description: 'Create a new email draft',
+			},
+			{
+				name: 'Create and Send',
+				value: 'send',
+				description: 'Create and send an email'
+			},
+			{
+				name: 'Reply',
+				value: 'reply',
+				description: 'Reply to an email',
+			},
+		],
+		default: 'create',
+		description: 'The operation to perform.',
+	},
+] as INodeProperties[];
+
+export const emailFields = [
+	{
+		displayName: 'Thread ID',
+		name: 'threadID',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'email'
+				],
+				operation: [
+					'reply'
+				]
+			},
+		},
+		placeholder: '172ce2c4a72cc243',
+		description: 'The id of the thread you are replying to.',
+	},
+	{
+		displayName: 'Message ID',
+		name: 'messageId',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'email'
+				],
+				operation: [
+					'reply'
+				]
+			},
+		},
+		placeholder: '<CAHNQoFsC6JMMbOBJgtjsqN0eEc+gDg2a=SQj-tWUebQeHMDgqQ@mail.gmail.com>',
```

Comment:
I haven't tested the node yet and don't know much about Message-IDs, but this placeholder seems to suggest that I should input `<` and `>` as well. Just commenting to make sure this is correct.